scripts/tablegen/tablegen.py

```diff
@@ -132,7 +136,13 @@ def decode_helm_yaml(s):
         print d
         if "<!-- AUTO-GENERATED-START -->" in d:
             break
+    # if os.path.exists("values.yaml"):
+    #     os.remove("values.yaml")
+    # else:
+    #     print("values.yaml doesn't exist")
 
+    # values = 'https://raw.githubusercontent.com/istio/istio/master/install/kubernetes/helm/istio/values.yaml'
```

Comment:
this is really cool!  I hadn't really considered this - I was instead thinking we would add a command line argument to pass the values in.  This is better.  We do need to be able to handle specific tags.  For the moment, its ok if this is manual, but perhaps the tag or branch could be passed in via cli?